dynamic-rapidpro-workspace-settings/package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "update-rapidpro-workspace-settings",
+  "version": "1.0.0",
+  "description": "dynamic-rapidpro-workspace Shared GitHub Action",
+  "main": "index.js",
+  "scripts": {
+    "eslint": "npx eslint {,**}/*.js {,**}/*.json",
```

Comment:
We probably should have the expressions quoted to prevent any interpretation from the underlying shell

Reply:
Gotcha